--- /home/qed/Projects/py/goo-engine-linux-wrapper/diff_ref/buildinfo.c.from	2026-08-15 22:04:34.617399028 -0400
+++ /home/qed/Projects/py/goo-engine-linux-wrapper/diff_ref/buildinfo.c.to	2026-08-15 22:04:34.619724401 -0400
@@ -51,5 +51,7 @@
 char build_linkflags[] = "unmaintained buildsystem alert!";
 char build_system[] = "unmaintained buildsystem alert!";
 #  endif
+int TIFFFaxBlackCodes;
+int TIFFFaxWhiteCodes;
 
 #endif  // BUILD_DATE
